### DISCUSSION TITLE
Api GitHub

### DISCUSSION THREAD

**Discussion by romulo2735** [frontendbr/forum#330] [Discussão]
Fui selecionado para fazer um Web App PHP na API do github, To meio perdido, queria um norte para eu conseguir implementar. Ele tem que ser simples pesquisar usuarios,listar e permitir detalhes dos usuarios como nome,foto ...

**Comment by romulo2735** [comment]
https://github.com/secaoweb/teste-vaga-estagio-php

**Comment by marcoaure** [comment]
Fala Romulo, blz?

cara faz uns requests ajax, utilizando a documentação do github…
eu fiz uma paradinha semelhante, utilizando somente javascript sem libs, mas usando  uma api de tempo digitando o nome da cidade…
Se você quiser um norte: https://github.com/cursosmau/advanced-javascript-finalproject https://github.com/cursosmau/advanced-javascript-finalproject

Abraço!

**Comment by romulo2735** [comment]
valeu

**Comment by romulo2735** [comment]
Li a documentação da API  e não entendi muita coisa.

**Comment by waldemirflj** [comment]
Hehehe..

Man, você manja de PHP?

2016-10-18 21:59 GMT-02:00 Romulo Sousa notifications@github.com:

**Comment by romulo2735** [comment]
Eu sei bem, mas não sou um jedi

**Comment by romulo2735** [comment]
é porque eu nunca tinha mexido com API,  talvez por isso a dificuldade.
